src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/CrdLicence.kt

```diff
@@ -250,6 +250,12 @@ class CrdLicence(
 
   override fun deactivate() = copy(statusCode = LicenceStatus.INACTIVE)
 
+  fun timeOut() = copy(
+    statusCode = LicenceStatus.TIME_OUT,
```

Comment:
can we call this TIMED_OUT ?

Reply:
I think you are the one who suggested this Andy.